MPAS/module_bl_mynnedmf_driver.F90

```diff
@@ -233,34 +230,25 @@ subroutine mynnedmf_driver &
     qbuoy,       &!
     qdiss         !
 
-#if(WRF_CHEM == 1)
 !--- input arguments for PBL and free-tropospheric mixing of chemical species:
```

Comment:
If the #if is removed, and no arrays (frp_mean, emis_ant_no, etc) are input into the scheme, will it still compile and run? I ask because I am supposed to make a PR to NCAR's repository soon.

Reply:
I think well have to use PRESENT checks to initialize the column arrays.

Reply:
Yes, it will still compile and run and passed our regression test runs. The results of regression test runs are posted in https://github.com/ufs-community/MPAS-Model/pull/121.

Reply:
Sure, I will use PRESENT checks to initialize the column arrays.

Reply:
> I think well have to use PRESENT checks to initialize the column arrays.

Done. Thanks.